indico/modules/events/abstracts/__init__.py

```diff
@@ -134,6 +134,11 @@ def _reviewing_area_visible(conf):
             return False
         return bool(get_user_tracks(conf.as_event, session.user))
 
+    def _submit_abstract_visible(conf):
+        if not session.user or not conf.has_feature('abstracts'):
```

Comment:
Maybe we should redirect to the login when `not session.user`? Otherwise it may look to non-indico users like they can't submit an abstract.